inst/@sym/private/elementwise_op.m

```diff
@@ -97,7 +97,10 @@
           'elements = []'
           'for i in range(0, len(q)):'
           '    elements.append(_op(*[k[i] if isinstance(k, MatrixBase) else k for k in _ins]))'
```

Comment:
I'm not sure what test hits this code on line 99 yet, but I think from doing #1226 that this will need to be replaced with either 2-d indexing or some kind of `flatten()`

Reply:
wait, I think I already did this in #1223...  :-X